src/commands/targets.rs

```diff
@@ -0,0 +1,197 @@
+use anyhow::{bail, Context, Result};
+use clap::Args;
+use std::{
+    fs,
+    path::{Path, PathBuf},
+};
+use wac_types::{ExternKind, ItemKind, Package, SubtypeChecker, Types, WorldId};
+
+/// Parses a WAC source file into a JSON AST representation.
+#[derive(Args)]
+#[clap(disable_version_flag = true)]
+pub struct TargetsCommand {
+    /// The path to the component
+    #[clap(value_name = "COMPONENT_PATH")]
+    pub component: PathBuf,
+    /// The path to the WIT definition
+    #[clap(value_name = "WIT_PATH")]
+    pub wit: PathBuf,
```

Comment:
Given these are heterogeneous positional arguments, I worry that users will constantly forget the ordering (à la `ln -s`).

Given that the only alternative I can think of is turning one (or both) of them into a require option (probably the wit file given we have the `--world` option that specifically works on the wit), which itself is not great UX, there's probably not much to do here.

Reply:
@rylev how do you feel about `--wit`. I know it's not a great alternative but i agree with Peter here, i personally with forget this in a week 😆